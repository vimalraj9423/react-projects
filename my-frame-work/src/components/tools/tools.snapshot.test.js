import {List,Menutools} from "./index";
import renderer from 'react-test-renderer';
import React from "react";

import { configure, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import toJson from 'enzyme-to-json';
configure({ adapter: new Adapter() }); 
it('renders correctly for menutools', () => {
    const tree = renderer
      .create(<Menutools/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly for List without props data', () => {

    const tree = renderer
      .create(<List/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });


  it('renders correctly for List with props list', () => {

    const tree = renderer
      .create(<List items={['item1','item2']}/>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders correctly for List with props list with enzyme', () => {

    const wrapper = shallow(<List items={['item1','item2']}/>);
    expect(toJson(wrapper)).toMatchSnapshot();
  });