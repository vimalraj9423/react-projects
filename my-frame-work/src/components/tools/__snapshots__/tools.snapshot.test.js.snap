// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly for List with props list 1`] = `
<ul
  className="list-items undefined "
>
  <li
    className="item"
  >
    item1
  </li>
  <li
    className="item"
  >
    item2
  </li>
</ul>
`;

exports[`renders correctly for List with props list with enzyme 1`] = `
<ul
  className="list-items undefined "
>
  <li
    className="item"
    key="item1"
  >
    item1
  </li>
  <li
    className="item"
    key="item2"
  >
    item2
  </li>
</ul>
`;

exports[`renders correctly for List without props data 1`] = `
<span
  className="empty-message"
>
  No items in list
</span>
`;

exports[`renders correctly for menutools 1`] = `
<a
  class="navicon-button x"
>
  <div
    className="navicon"
  />
</a>
`;
